NoSpan in console

## Minimized example
```scala
sbt:dotty-simple> console
[info] Fetching artifacts of 
https://repo1.maven.org/maven2/ch/epfl/lamp/dotty-compiler_0.25/0.25.0-bin-20200604-2fbefb0-NIGHTLY/dotty-compiler_0.25-0.25.…
  100.0% [##########] 11.9 MiB (104.6 KiB / s)
[info] Fetched artifacts of 
[info] Compiling 13 Scala sources to /Users/vletal/Workspace/Personal/loonyssh/target/scala-0.25/classes ...

scala> enum E:
     |   case A
     | 
[error] (run-main-0) java.lang.AssertionError: assertion failed
[error] java.lang.AssertionError: assertion failed
[error]         at dotty.DottyPredef$.assertFail(DottyPredef.scala:16)
[error]         at dotty.tools.dotc.util.Spans$Span$.point$extension(Spans.scala:56)
[error]         at dotty.tools.dotc.util.SourcePosition.point(SourcePosition.scala:23)
[error]         at dotty.tools.dotc.util.SourcePosition.line(SourcePosition.scala:25)
[error]         at dotty.tools.repl.ReplDriver.$anonfun$5(ReplDriver.scala:258)
[error]         at scala.math.Ordering$$anon$1.compare(Ordering.scala:140)
[error]         at java.base/java.util.TimSort.countRunAndMakeAscending(TimSort.java:355)
[error]         at java.base/java.util.TimSort.sort(TimSort.java:220)
[error]         at java.base/java.util.Arrays.sort(Arrays.java:1441)
[error]         at scala.collection.SeqOps.sorted(Seq.scala:703)
[error]         at scala.collection.SeqOps.sorted$(Seq.scala:695)
[error]         at scala.collection.immutable.List.scala$collection$immutable$StrictOptimizedSeqOps$$super$sorted(List.scala:79)
```

--- a/compiler/src/dotty/tools/dotc/util/SourcePosition.scala
+++ b/compiler/src/dotty/tools/dotc/util/SourcePosition.scala
@@ -86,6 +86,8 @@ extends SrcPos, interfaces.SourcePosition, Showable {
 
 /** A sentinel for a non-existing source position */
 @sharable object NoSourcePosition extends SourcePosition(NoSource, NoSpan, null) {
+  override def line: Int = -1
+  override def column: Int = -1
   override def toString: String = "?"
   override def withOuter(outer: SourcePosition): SourcePosition = outer
 }

--- a/compiler/test/dotty/tools/repl/ReplCompilerTests.scala
+++ b/compiler/test/dotty/tools/repl/ReplCompilerTests.scala
@@ -233,3 +233,31 @@ object ReplCompilerTests {
   }
 
 }
+
+class ReplXPrintTyperTests extends ReplTest(ReplTest.defaultOptions :+ "-Xprint:typer") {
+  @Test def i9111 = fromInitialState { implicit state =>
+    run("""|enum E {
+           |  case A
+           |}""".stripMargin)
+    assert(storedOutput().trim().endsWith("// defined class E"))
+  }
+
+  @Test def i10883 = fromInitialState { implicit state =>
+    run("val a = 42")
+    assert(storedOutput().trim().endsWith("val a: Int = 42"))
+  }
+}
+
+class ReplVerboseTests extends ReplTest(ReplTest.defaultOptions :+ "-verbose") {
+  @Test def i9111 = fromInitialState { implicit state =>
+    run("""|enum E {
+           |  case A
+           |}""".stripMargin)
+    assert(storedOutput().trim().endsWith("// defined class E"))
+  }
+
+  @Test def i10883 = fromInitialState { implicit state =>
+    run("val a = 42")
+    assert(storedOutput().trim().endsWith("val a: Int = 42"))
+  }
+}

--- a/compiler/test/dotty/tools/repl/ReplTest.scala
+++ b/compiler/test/dotty/tools/repl/ReplTest.scala
@@ -16,19 +16,8 @@ import dotty.tools.dotc.reporting.MessageRendering
 import org.junit.{After, Before}
 import org.junit.Assert._
 
-
-class ReplTest(withStaging: Boolean = false, out: ByteArrayOutputStream = new ByteArrayOutputStream) extends ReplDriver(
-  Array(
-    "-classpath",
-    if (withStaging)
-      TestConfiguration.withStagingClasspath
-    else
-      TestConfiguration.basicClasspath,
-    "-color:never",
-    "-Yerased-terms",
-  ),
-  new PrintStream(out, true, StandardCharsets.UTF_8.name)
-) with MessageRendering {
+class ReplTest(options: Array[String] = ReplTest.defaultOptions, out: ByteArrayOutputStream = new ByteArrayOutputStream)
+extends ReplDriver(options, new PrintStream(out, true, StandardCharsets.UTF_8.name)) with MessageRendering {
   /** Get the stored output from `out`, resetting the buffer */
   def storedOutput(): String = {
     val output = stripColor(out.toString(StandardCharsets.UTF_8.name))
@@ -103,3 +92,8 @@ class ReplTest(withStaging: Boolean = false, out: ByteArrayOutputStream = new By
     end if
   }
 }
+
+object ReplTest:
+  val commonOptions = Array("-color:never", "-Yerased-terms")
+  val defaultOptions = commonOptions ++ Array("-classpath", TestConfiguration.basicClasspath)
+  lazy val withStagingOptions = commonOptions ++ Array("-classpath", TestConfiguration.withStagingClasspath)

--- a/staging/test/scala/quoted/staging/repl/StagingScriptedReplTests.scala
+++ b/staging/test/scala/quoted/staging/repl/StagingScriptedReplTests.scala
@@ -8,7 +8,7 @@ import org.junit.Test
 import org.junit.experimental.categories.Category
 
 /** Runs all tests contained in `staging/test-resources/repl-staging` */
-class StagingScriptedReplTests extends ReplTest(withStaging = true) {
+class StagingScriptedReplTests extends ReplTest(ReplTest.withStagingOptions) {
 
   @Category(Array(classOf[BootstrappedOnlyTests]))
   @Test def replStagingTests = scripts("/repl-staging").foreach(testFile)

